client/cpp/kserve-api/README.md

```diff
@@ -0,0 +1,437 @@
+# KServe API usage samples
+
+OpenVINO Model Server 2022.2 release introduced support for [KServe API](https://github.com/kserve/kserve/tree/master/docs/predict-api/v2).
```

Comment:
KServe was introduced in 2022.2 but some examples will work only with 2022.3 included raw format extension